homeassistant/components/modbus/switch.py

```diff
@@ -154,6 +154,8 @@ def turn_off(self, **kwargs):
     def update(self):
         """Update the entity state."""
         if not self._verify_active:
+            self._available = True
```

Comment:
I'm not sure if this code gets called. First call to `update()` is scheduled like this:

```python
        if self._verify_active:
            async_track_time_interval(
                self.hass, lambda arg: self.update(), self._scan_interval
            )
```

Reply:
Four eyes are better than two :-) I had a feeling of having forgotten something. The "if" is removed.

Or is your preference, not to make the switch unavailable for a short while ?

Reply:
No, your solution is OK. I'll check the code locally.